GAE/src/locale/fr.properties

```diff
@@ -396,11 +399,10 @@ Save\ question = Enregistrer la question
 Save\ translations = Enregistrer les traductions
 Save\ user\ info = Enregistrer les infos utilisateur
 Saving = Enregistrement
-Search\ datapoint\ by\ name\ or\ ID = Cherchez des points de donn�es par nom ou ID
+Search\ datapoint\ by\ name\ or\ ID,\ or\ press\ Enter\ to\ show\ all = Cherchez des points de donn�es par nom ou ID
```

Comment:
Oups this one should be: Cherchez des points de données par nom ou ID ou appuyez sur Entrer pour tout afficher

Reply:
I have changed it in transifex too